students/temu4/DimaTimoschenko02/hw1/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        let age = prompt('enter your age: ')
+        while(age < 18){
+            age = prompt('enter your age: ')
+        }
```

Comment:
Preferably to use `do...while` loop here 
```suggestion
       let age;
       do {
            age = prompt('enter your age: ')
        } while(age < 18)
```

Reply:
ok, i just wrote some code and didnt think much)